docs/03/a.md

```diff
@@ -18,11 +18,13 @@ The formatting options are:
     decap    | decapitalize    : lowercase first letter
     start    | start-case      : uppercase the first letter of each word
     word     | word-only       : remove all non-word letters (only a-zA-Z0-9_)
+    words    | words-only      : replace all non-word letters (only a-zA-Z0-9) with a whitespace
```

Comment:
I think `word` vs `words` is too subtle. How about `space`?

Reply:
that's fair. what about `sentence`?

Reply:
Doesn't sentence imply some sort of grammatical structure? Turning "My-Example-Project" into "My Example Project" is not exactly sentencifying things.